examples/README.md

```diff
@@ -0,0 +1,36 @@
+# Examples
+
+This folder contains examples of how to use the `librespot` library for various purposes.
+
+## How to run the examples
+
+In general, to invoke an example, clone down the repo and use `cargo` as follows:
+
+```
+cargo run --example [filename]
+``` 
+
+in which `filename` is the file name of the example, for instance `get_token` or `play`.
+
+### Acquiring an access token
+
+Most examples require an access token as the first positional argument. **Note that an access token
+gained by the client credentials flow will not work**. `librespot-oauth` provides a utility to 
+acquire an access token using an OAuth flow, which will be able to run the examples. To invoke this, 
+run:
+
+```
+cargo run --package librespot-oauth --example oauth_sync
+```
+
+A browser window will open and prompt you to authorize with Spotify. Once done, take the 
+`access_token` property from the dumped object response and proceed to use it in examples. You may
+find it convenient to save it in a shell variable like `$ACCESS_TOKEN`.
+
+Once you have obtained the token you can proceed to run the example. Check each individual
+file to see what arguments are expected. As a demonstration, here is how to invoke the `play` 
+example to play a song -- the second argument is the URI of the track to play.
+
+```
+cargo run --example play "$ACCESS_TOKEN" 2WUy2Uywcj5cP0IXQagO3z
```

Comment:
This URI is for https://open.spotify.com/track/2WUy2Uywcj5cP0IXQagO3z, I'm not sure if we have to be careful about 'featuring' tracks like this, surely it should be OK from a copyright perspective if the user has to provide their own token and therefore has rights to stream the music?

Reply:
I think this is the least part we need to worry about xD. But even so we already reference some songs or context inside our examples and or tests. So this should probably be fine.